/*
 *  LDForge: LDraw parts authoring CAD
 *  Copyright (C) 2013 Santeri Piippo
 *
 *  This program is free software: you can redistribute it and/or modify
 *  it under the terms of the GNU General Public License as published by
 *  the Free Software Foundation, either version 3 of the License, or
 *  (at your option) any later version.
 *
 *  This program is distributed in the hope that it will be useful,
 *  but WITHOUT ANY WARRANTY; without even the implied warranty of
 *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 *  GNU General Public License for more details.
 *
 *  You should have received a copy of the GNU General Public License
 *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */

act (New)
act (NewFile)
act (Open)
act (DownloadFrom)
act (Save)
act (SaveAs)
act (SaveAll)
act (Close)
act (CloseAll)
act (InsertFrom)
act (ExportTo)
act (Settings)
act (SetLDrawPath)
act (ScanPrimitives)
act (Exit)
act (ResetView)
act (Axes)
act (Wireframe)
act (BFCView)
act (SetOverlay)
act (ClearOverlay)
act (Screenshot)
act (InsertRaw)
act (NewSubfile)
act (NewLine)
act (NewTriangle)
act (NewQuad)
act (NewCLine)
act (NewComment)
act (NewBFC)
act (NewVertex)
act (Undo)
act (Redo)
act (Cut)
act (Copy)
act (Paste)
act (Delete)
act (SelectAll)
act (SelectByColor)
act (SelectByType)
act (ModeDraw)
act (ModeSelect)
act (SetDrawDepth)
act (SetColor)
act (Autocolor)
act (Uncolorize)
act (Inline)
act (InlineDeep)
act (Invert)
act (MakePrimitive)
act (SplitQuads)
act (EditRaw)
act (Borders)
act (CornerVerts)
act (RoundCoordinates)
act (Visibility)
act (ReplaceCoords)
act (Flip)
act (Demote)
act (Ytruder)
act (Rectifier)
act (Intersector)
act (Isecalc)
act (Coverer)
act (Edger2)
act (Help)
act (About)
act (AboutQt)
act (GridCoarse)
act (GridMedium)
act (GridFine)
act (Edit)
act (MoveUp)
act (MoveDown)
act (MoveXNeg)
act (MoveXPos)
act (MoveYNeg)
act (MoveYPos)
act (MoveZNeg)
act (MoveZPos)
act (RotateXNeg)
act (RotateXPos)
act (RotateYNeg)
act (RotateYPos)
act (RotateZNeg)
act (RotateZPos)
act (RotationPoint)
act (AddHistoryLine)

#undef act